packages/react-ui/components/Link/__tests__/Link-test.tsx

```diff
@@ -1,61 +1,62 @@
-import { mount } from 'enzyme';
 import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import { LinkProps } from '..';
-import { Link } from '../Link';
+import { Link, LinkDataTids } from '../Link';
 
-const render = (props?: LinkProps) => mount(<Link {...props} />);
+const renderRTL = (props?: LinkProps) => render(<Link {...props} />);
 
 describe('Link', () => {
   it('calls `onClick` when link clicked', () => {
     const onClick = jest.fn();
 
-    const wrapper = render({ onClick });
-    wrapper.find('a').simulate('click');
-
+    renderRTL({ onClick });
+    userEvent.click(screen.getByTestId(LinkDataTids.root));
     expect(onClick).toHaveBeenCalled();
   });
 
   describe('disabled link', () => {
     it('does not call `onClick` when link clicked', () => {
       const onClick = jest.fn();
 
-      const wrapper = render({ onClick, disabled: true });
-      wrapper.find('a').simulate('click');
+      renderRTL({ onClick, disabled: true });
+
+      const linkElement = screen.getByTestId(LinkDataTids.root);
+      linkElement.style.pointerEvents = 'auto';
+      userEvent.click(linkElement);
```

Comment:
Это довольно большой костыль. Но проблему можно официально обойти.

```suggestion
      userEvent.click(linkElement, {}, { skipPointerEventsCheck: true });
```